patches/master_patch.patch

```diff
@@ -1010,6 +1010,42 @@ index 1c68e13912c2dfa7cc842bab169029c58677811a..3731fe3df76b0ac1dbf155c221fc64e2
  }
  
  content::WebContents* GuestViewManager::GetGuestByInstanceIDSafely(
+diff --git a/components/history/core/browser/history_service.cc b/components/history/core/browser/history_service.cc
```

Comment:
this is the only diff that should have been added. the other diffs in this file were generated by `npm run update_patches` prior to my changes. you can ignore 5466f740ca9144c7c261ab26f3c429683d71d7da if those other diffs are not needed.